src/components/SeriesSearch/SeriesSearchForm.js

```diff
@@ -31,27 +31,34 @@ export const SeriesSearchForm = () => {
           onChange={(event) => onHandleChange(event.target.value)}
         />
       </form>
-      <div id="search-results-overlay">
+      <div className="search-results-overlay">
         {_.map(searchResults, (result, index) => {
           const {
             imageURL,
             numberOfEpisodes,
             title,
             titleID,
             seriesStartYear,
-            seriesEndYear,
+            seriesEndYear = 'Present',
           } = result;
 
           return (
-            <div key={index}>
+            <div className="search-series-suggestion" key={index}>
               <Link to={`chart/${titleID}`}>
-                <p>
-                  {title} ({numberOfEpisodes})
-                </p>
-                <p>
-                  {seriesStartYear}-{seriesEndYear}
-                </p>
-                <img src={imageURL} width={50} />
+                <img
+                  className="search-series-img"
+                  alt="tv show poster image"
+                  src={imageURL}
+                  width={50}
```

Comment:
this width property probably should not be set on this element. maybe add it to the class